adele-payment-manager/src/test/java/com/microservicesteam/adele/payment/PaymentManagerTest.java

```diff
@@ -86,6 +85,7 @@ public void executePaymentShouldReturnWithApprovedResponseWhenExecutedPaymentHas
         Payment executedPayment = new Payment();
         executedPayment.setId("paymentId");
         executedPayment.setState("approved");
+        when(executePaymentRequestMapper.mapTo(any(ExecutePaymentRequest.class))).thenReturn(new Payment());
```

Comment:
This is needed because `Mockito.any(Class clazz)` doesn't match on `null` values anymore 😞